File: Chapter 2/password_strength_checker.py
password = input("Enter Password: ")

password_length = len(password) 

if password_length > 6 and password_length <= 10:
	print("Medium")
elif password_length < 6:
	print("Weak")
elif password_length > 10:
	print("Strong")
elif password_length < 1:
	print("Invalid")

File: Chapter 2/PasswordStrengthChecker.java
import java.util.Scanner;

public class PasswordStrengthChecker {

public static void main (String[]args) {

Scanner input = new Scanner(System.in);

System.out.print("Enter Password :");

String password = input.nextLine();

int length = password.length();

if (length > 6 && length <=10) {

System.out.print("Medium");

} else if (length < 6) {

System.out.print("Weak");

} else if (length > 10) {

System.out.print("Strong");

} else if (length < 1) {

System.out.print("Invalid");

}

}

}

